src/stream.iced

```diff
@@ -0,0 +1,20 @@
+stream = require('stream');
+enc = require('./encoding');
+
+exports.StreamEncoder = class StreamEncoder extends stream.Transform
+
+	constructor : (@encoder) ->
+		super({highWaterMark : Math.floor(4096/encoder.in_block_len)})
```

Comment:
can you pull out this `4096` block specification into a constant both classes share?

And actually, my formula is totally whack, and I don't see how the tests were passing with it.  It should be:

```
encoder.in_block_len * Math.ceil(desired_block_size / encoder.in_block.len)
```

Sorry I gave you the wrong formula